engine/src/main/java/org/terasology/engine/rendering/nui/layers/mainMenu/advancedGameSetupScreen/AdvancedGameSetupScreen.java

```diff
@@ -509,12 +525,12 @@ public Boolean get() {
                 getManager().createScreen(MessagePopup.ASSET_URI, MessagePopup.class).
                         setMessage("Error", "Game seed cannot be empty!");
             } else {
-                universeWrapper.setSeed(seed.getText());
+                context.get(UniverseWrapper.class).setSeed(seed.getText());
```

Comment:
We're fetching the universe 3 times from the context here. Can't we do that once and then use that object?

```suggestion
                UniverseWrapper universe = context.get(UniverseWrapper.class);
                universe.setSeed(seed.getText());
```